docs/pages-for-subheaders/rke2-hardening-guide.md

```diff
@@ -64,22 +65,6 @@ sudo useradd -r -c "etcd user" -s /sbin/nologin -M etcd -U
 
 The runtime requirements to pass the CIS Benchmark are centered around pod security, network policies and kernel parameters. Most of this is automatically handled by RKE2 when using a valid `cis-1.xx` profile, but some additional operator intervention is required. These are outlined in this section.
 
-### Ensure `protect-kernel-defaults` is set
```

Comment:
Same as before:

> 

This needs to be inside a specific tab depending on the CIS benchmark version, because it's still needed in CIS 1.23 and CIS 1.24.